tests/virtual_chassis/topology.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
standalone-chassis.json

Reply:
renaming to chassis_standalone.json as other topo files are also starting with chassis_* prefix.